### PR TITLE
Initialize schema types

### PR DESCRIPTION
**Pull request by keithradford**
Initialized all the schema types in specific folders for everything we need. Nexus has an SDL converter (https://nexusjs.org/converter) so I just copy and pasted the schema we made with company 3 into there. I split everything up into respective files just for organization, but otherwise it was all automated.

`./schema.graphql` should now match `./test.graphql` (schema is auto-generated by nexus and test is what we made with company 3).

### CODE REVIEW

File: src/types/Schedule.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { arg, extendType, inputObjectType, intArg, nonNull, objectType } from 'nexus';
+import { CourseSection } from './Course/Section';
+import { Date } from './Date';
+import { Response } from './Response';
+import { Term } from './Term';
+
+export const GenerateScheduleInput = inputObjectType({
```
Comment (szeckirjr) [inline src/types/Schedule.ts:7]: soo just so I understand it better, if frontend team calls this endpoint and sends a term, **backend** will call the algorithm to generate schedule?

Comment (keithradford) [reply src/types/Schedule.ts:7]: Most likely, yes. I imagine algorithm will just expose an API endpoint or something.

File: schema.graphql [hunk 2]
```diff
@@ -2,6 +2,225 @@
...
+}
+
+type CourseID {
+  """Course code, e.g. 499, 310"""
+  code: String!
+
+  """Course subject, e.g. SENG, CSC"""
+  subject: String!
+
+  """Term course is offered in"""
```
Comment (RcrossF) [inline schema.graphql:29]: An example could be helpful like
```suggestion
  """Term course is offered in eg. 202201"""
```

**Comment by keithradford** [comment]
@szeckirjr @RcrossF  Let's keep note of all these comments (even the minor typo things) and bring them up with company 3 so we can make changes to the shared schema

**Review by RcrossF — APPROVED** [review]

**Pull request merged**